docs/rules/attribute-denylist.md

```diff
@@ -0,0 +1,26 @@
+# Disallows a set of attributes from being used (attribute-denylist)
```

Comment:
NIT: not a huge fan of "denylist" for wording and rule name

Reply:
was trying to be consistent with eslint itself, as it has `id-denylist` which seemed the most similar rule.

we could possibly call it `ban-attributes` since typescript-eslint has `ban-types`?

Reply:
Wasn't aware of the id-denylist rule. I guess based on that I'm okay with it. I think I have a slight preference to ban-attributes but either is good with me.